src/components/dashboard/GovernanceSection/GovernanceSection.tsx

```diff
@@ -137,6 +137,16 @@ const GovernanceSection = () => {
 // Prevent `GovernanceSection` hooks from needlessly being called
 const GovernanceSectionWrapper = () => {
   const chainId = useChainId()
+  const [isLayoutReady, setIsLayoutReady] = useState(false)
+
+  useEffect(() => {
+    setIsLayoutReady(true)
+  }, [])
+
+  if (!isLayoutReady) {
+    return null
+  }
+
```

Comment:
nit: Another way of doing this is using the `dynamic` import feature from nextjs

```
const GovernanceSectionWrapper = () => {
  const chainId = useChainId()
  if (!getSafeTokenAddress(chainId)) {
    return null
  }

  const DynamicGovernanceSection = dynamic(() => import('./GovernanceSection'))

  return <DynamicGovernanceSection />
}
```